lib/params.js

```diff
@@ -52,10 +52,20 @@ function getIndentationFromStylelintRules (rules) {
 }
 
 function loadConfig (stylelint, file, configFile) {
-  return stylelint.getConfigForFile(file, configFile).then(function (stylelintrc){
-    return stylelintrc && stylelintrc.config && stylelintrc.config.rules
+  return Promise.resolve().then(function (){
+    if (file && !stylelint._options.ignoreDisables) {
```

Comment:
in my opinion, we don't have to use the internal option of stylelint. what do you think about it?